charts/app-config/values-integration.yaml

```
@@ -26,6 +27,8 @@ govukApplications:
         alb.ingress.kubernetes.io/group.name: assets-origin
         alb.ingress.kubernetes.io/group.order: '10'
         alb.ingress.kubernetes.io/load-balancer-attributes: access_logs.s3.enabled=true,access_logs.s3.bucket=govuk-integration-aws-logging,access_logs.s3.prefix=elb/assets-origin-eks
+        alb.ingress.kubernetes.io/conditions.rule-{{ .Release.Name }}: >
+          [{"field":"host-header","hostHeaderConfig":{"values":["{{ .Values.assetsDomain }}"]}}]
```

Since this is going through `toYaml` already, might be cleaner to write it as YAML and use `toJson` (it's supposed to end up as JSON, right?)

```suggestion
        alb.ingress.kubernetes.io/conditions.rule-{{ .Release.Name }}: |
          - field: host-header
            hostHeaderConfig:
              values:
                - '{{ .Values.assetsDomain }}'
```

Alternatively could keep it as JSON here but prettify it, though I think it looks marginally clearer as YAML 🤷

```
[
  {
    "field": "host-header",
    "hostHeaderConfig": {
      "values": [
        "{{ .Values.assetsDomain }}"
      ]
    }
  }
]
```

---

As per the reply below - but the value has to be JSON. The `toJson` template method doesn't format this because its seen as just a multiline string.